circuit_maintenance_parser/output.py

```diff
@@ -164,34 +172,44 @@ class Maintenance(BaseModel, extra=Extra.forbid):
 
     def __init__(self, **data):
         """Initialize the Maintenance object."""
-        self._metadata = data.pop("_metadata")
+        metadata = data.pop("_metadata")
         super().__init__(**data)
+        self._metadata = metadata
 
-    # pylint: disable=no-self-argument
-    @validator("status")
+    @field_validator("status")
+    @classmethod
     def validate_status_type(cls, value):
         """Validate Status type."""
         if value not in Status:
             raise ValueError("Not a valid status type")
         return value
 
-    @validator("provider", "account", "maintenance_id", "organizer")
+    @field_validator("provider", "account", "maintenance_id", "organizer")
+    @classmethod
     def validate_empty_strings(cls, value):
         """Validate emptry strings."""
         if value in ["", "None"]:
             raise ValueError("String is empty or 'None'")
         return value
 
-    @validator("circuits")
+    # TODO[pydantic]: We couldn't refactor the `validator`, please replace it by `field_validator` manually.
+    # Check https://docs.pydantic.dev/dev-v2/migration/#changes-to-validators for more information.
```

Comment:
What's TODO here from our end?

Reply:
sorry, I had a local commit to remove them that I forgot to push